code/modules/mob/living/simple_animal/abnormality/aleph/melting_love.dm

```diff
@@ -213,12 +229,20 @@
 	resize = 1.3
 	update_transform()
 
+/mob/living/simple_animal/hostile/slime/big/CanAttack(atom/the_target)
+    if(isliving(target) && !ishuman(target))
+        var/mob/living/L = target
+        if(L.stat == DEAD)
+            return FALSE
+    return ..()
+
 /mob/living/simple_animal/hostile/slime/big/AttackingTarget()
```

Comment:
Same here.

Reply:
I can also delete the initialize ? also do I need to leave in slime 	
robust_searching = TRUE
stat_attack = DEAD